docs/parameters.md

```diff
@@ -0,0 +1,234 @@
+# Passing parameters to the brokers
```

Comment:
'The' here is used as a definite article but I think brokers is more of an abstract term, see [this link](http://www.ef-australia.com.au/english-resources/english-grammar/definite-article/)